daemon/cluster/metrics.go

```diff
@@ -0,0 +1,23 @@
+package cluster
+
+import "github.com/docker/go-metrics"
+
+var (
+	clusterInfo     metrics.LabeledGauge
+	clusterNodes    metrics.Gauge
+	clusterManagers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "daemon", nil)
+
+	clusterInfo = ns.NewLabeledGauge("swarm", "The information related to the cluster", metrics.Unit("info"),
```

Comment:
@aaronlehmann I'll update it. Thanks for the feedback. 

@stevvooe, I think it could also live there. But I'll defer this decision to you guys. We can always move that later.